_candidates/ken-houston.md

```diff
@@ -1,17 +1,17 @@
 ---
-id: 41
+id: 33
 name: Ken Houston
 photo_url: null
 website_url: null
-twitter_url: '@KHouston4Mayor'
+twitter_url: KHouston4Mayor
 votersedge_url: null
 first_name: Ken
 last_name: Houston
-ballot_item: 9
-office_election: 9
+ballot_item: 10
+office_election: 10
 bio: null
-committee_name: null
-is_accepted_expenditure_ceiling: null
+committee_name: No committee registered
```

Comment:
Can we just remove this from the spreadsheet? It's better if it shows up as `null`. Also, I'm not sure we're using this? We can just grab the committee from the filer id.